tests/js/JSHintBearTest.py

```diff
@@ -63,11 +72,11 @@
 JSHintBearDeprecationTest = verify_local_bear(
     JSHintBear,
     valid_files=(),
-    invalid_files=(test_es6,),
-    settings={"es_version": 3})
+    invalid_files=(test_es6, test_strictness),
+    settings={"es_version": 3, "javascript_strictness": True})
```

Comment:
maybe use the deprecated arguments here in general? Might that fix the coverage issue? :/